source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
+
+
+Avaliable timeouts
+------------------
+
+pfSense's captive portal has 3 integrated timeout options : 
+
+- **"Idle timeout"** causes users to be automatically disconnected after an amount of time of network inactivity.
```

Comment:
This sentence is a bit awkward, this is better:
`Users will be automatically disconnected after a defined amount of network inactivity.`